src/Components/AuthDialog/Components/AuthDialogSignUpPlaceholder.tsx

```diff
@@ -42,10 +42,14 @@ export const AuthDialogSignUpPlaceholder: FC = () => {
           Already have an account? Log in.
         </SkeletonText>
 
-        <SkeletonText variant="xs" textAlign="center">
-          By clicking Sign Up or Continue with Apple, Google, or Facebook, you
-          agree to Artsy’s Terms of Use and Privacy Policy and to receiving
-          emails from Artsy.
+        <SkeletonText
+          variant="xs"
+          textAlign="center"
+          data-testid="skeleton-disclaimer"
+        >
+          By clicking Sign Up or Continue with Email, Apple, Google, or
```

Comment:
**nitpick:** Not sure if it was raised before to the writing team, but, should `Sign Up` be `"Sign Up"`?
I was looking at the GitHub Terms and Conditions, and they were using some of the quotation [here](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#l-cancellation-and-termination) for the "internal" features. (I definitely used it incorrectly in my review comments 😅 )

Reply:
I don't think that quotations are needed in this context, especially since we'd need to put "Continue with Email" and all its variations in quotations as well.

Regarding the GitHub Terms and Conditions, I see them putting words or phrases in quotations when they need to be defined, like [here](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#a-definitions). That is commonly done in legal documents to make them easier to read, but I don't think we need to do that here.